lib/browser-runtime.ts

```diff
@@ -775,6 +780,31 @@ function beforeHandler(context: CompositionContext) {
   taskSpecEnvelopes(context);
 }
 
+function beforeAllHandler(this: Mocha.Context, context: CompositionContext) {
+  const { registry } = context;
+  let beforeAllHooks = registry.resolveBeforeAllHooks();
+  for(const beforeAllHook of beforeAllHooks) {
+      if (beforeAllHook) {
+        const hook = beforeAllHook;
+        cy.then(() => {
+          const start = createTimestamp();
+          return cy.wrap(start, { log: false });
+        })
+          .then((start) => {
+            runStepWithLogGroup({
+              fn: () => registry.runHook(this, hook),
+              keyword: "BeforeAll"
+            });
+
+            return cy.wrap(start, { log: false });
+          })
+          .then((start) => {
+            const end = createTimestamp();
+          });
+      }
```

Comment:
Wrapping `runStepWithLogGroup` in order to obtain start- & end-time is only necessary when these values are actually used for something and this isn't the case here. Thus, you can replace 

```js
cy.then(() => {
  const start = createTimestamp();
  return cy.wrap(start, { log: false });
})
  .then((start) => {
    runStepWithLogGroup({
      fn: () => registry.runHook(this, hook),
      keyword: "BeforeAll"
    });

    return cy.wrap(start, { log: false });
  })
  .then((start) => {
    const end = createTimestamp();
  });
```

.. with

```js
runStepWithLogGroup({
  fn: () => registry.runHook(this, hook),
  keyword: "BeforeAll"
});
```

Reply:
I see. Made this change